kube_metrics_server/README.md

```diff
@@ -8,15 +8,28 @@ This check monitors [Kube_metrics_server][1] v0.3.0+, a component used by the Ku
 
 ### Installation
 
-The Kube_metrics_server check is included in the [Datadog Agent][2] package.
-No additional installation is needed on your server.
+The Kube_metrics_server check is included in the [Datadog Agent][2] package. No additional installation is needed on your server.
 
 ### Configuration
 
+#### Host
+
+Follow the instructions below to configure this check for an Agent running on a host. For containerized environments, see the [Containerized](#containerized) section.
+
 1. Edit the `kube_metrics_server.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your kube_metrics_server performance data. See the [sample kube_metrics_server.d/conf.yaml][2] for all available configuration options.
 
 2. [Restart the Agent][3].
 
+#### Containerized
+
+For containerized environments, see the [Kubernetes Autodiscovery Integration Templates][4] for guidance on applying the parameters below.
+
+| Parameter            | Value                                                |
+| -------------------- | ---------------------------------------------------- |
+| `<INTEGRATION_NAME>` | `prometheus`                                         |
```

Comment:
shouldn't be `kube_metrics_server` ?

Reply:
it seems that it's the prometheus / open-metrics check that is performed to collect the data.

Reply:
yes `kube_metrics_server` is  a prometheus/open-metrics based integration, you can collect the same metrics with the prometheus/openmetrics checks but they'll be considered custom metrics and they won't populate the OOTB dashboard of the kube_metrics_server  integration

Reply:
indeed
```suggestion
| `<INTEGRATION_NAME>` | `kube_metrics_server `                                         |
```